apps/web/src/components/SearchAddressInput/index.tsx

```diff
@@ -31,12 +29,13 @@ export default function SearchAddressInput({ onChange }: SearchAddressInputProps
   /* 2. User enters an Basename */
   const validBasename = isBasename(value);
 
-  const { basenameChain } = useBasenameChain(value as BaseName);
```

Comment:
when `value` is an address or L1 ens and not a basename this always returns sepolia. `value` the address or name the user is trying to send to.